synapse/config/oidc_config.py

```diff
@@ -205,6 +206,8 @@ def generate_config_section(self, config_dir_path, server_name, **kwargs):
     "type": "object",
     "required": ["issuer", "client_id", "client_secret"],
     "properties": {
+        "idp_id": {"type": "string"},
```

Comment:
Looks like json schema supports `minLength` / `maxLength` parameters that we could possible use?

Reply:
mmm that might be better.